scripts/piper/generate.py

```
@@ -1556,14 +1616,21 @@ def get_sw_models():
 
     for m in tr_TR:
         m.lang = "tr_TR"
 
-    ans = tr_TR
+    for m in ku_TR:
+        m.lang = "ku_TR"
+
+    ans = tr_TR + ku_TR
 
     for m in ans:
         m.text = "Bir evin duvarları, bir adamın sözü, bir kadının gülü kırılmaz"
```

Ruff (0.15.11)

[warning] String contains ambiguous `ı` (LATIN SMALL LETTER DOTLESS I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)